components/consumers/pdf/Dockerfile

```diff
@@ -1,11 +1,11 @@
-FROM golang:latest
+FROM golang:1.22-bookworm
 
-WORKDIR /playwright
-RUN go mod init github.com/smithy-security/pdf-consumer && \
-        go get -u github.com/playwright-community/playwright-go && \
-        go run github.com/playwright-community/playwright-go/cmd/playwright@latest install --with-deps
+# Install playwright cli with right version for later use
+RUN go install github.com/playwright-community/playwright-go/cmd/playwright@v0.4702.0 && \
+    apt-get update && apt-get install -y ca-certificates tzdata &&\
+    playwright install chromium --with-deps
 
-ENV PATH="${PATH}:/go/pkg/mod/github.com/playwright-community"
-COPY components/consumers/pdf/pdf /playwright/pdf
-COPY components/consumers/pdf/default.html /playwright/default.html
-ENTRYPOINT ["/playwright/pdf"]
+COPY components/consumers/pdf/linux/amd64/pdf /application/pdf
```

Comment:
Can we customise this path based on `GOOS` and `GOARCH` set by the user?

There are a few examples on the repo